### PR TITLE
Add Bitcoin EVM

### PR DESCRIPTION
**Pull request by BitcoinEvm**
Added Support For Bitcoin EVM Chain ID 2203

### CODE REVIEW

**Review by ligi — CHANGES_REQUESTED** [review]
the icon in iconsDownload cannot have the .PNG ending

**Comment by BitcoinEvm** [comment]
Changes made @ligi PNG removed

File: _data/icons/ebtc.json [hunk 1]
```diff
@@ -0,0 +1,8 @@
+[
+  {
+    "url":"ipfs://bafkreic4aq265oaf6yze7ba5okefqh6vnqudyrz6ovukvbnrlhet36itle",
+    "width":896,
```
Comment (ligi) [inline _data/icons/ebtc.json:4]: size seems to be 200x200
✓ ~/tmp ipfs get bafkreic4aq265oaf6yze7ba5okefqh6vnqudyrz6ovukvbnrlhet36itle
Saving file(s) to bafkreic4aq265oaf6yze7ba5okefqh6vnqudyrz6ovukvbnrlhet36itle
 33.39 KiB / 33.39 KiB [============================================] 100.00% 0s
✓ ~/tmp identify bafkreic4aq265oaf6yze7ba5okefqh6vnqudyrz6ovukvbnrlhet36itle
bafkreic4aq265oaf6yze7ba5okefqh6vnqudyrz6ovukvbnrlhet36itle PNG 200x200 200x200+0+0 8-bit sRGB 34195B 0.000u 0:00.000

**Review by ligi — CHANGES_REQUESTED** [review]
size is wrong - also the file in iconsDownload does not match the CID - please just remove

**Comment by BitcoinEvm** [comment]
Ok It has been removed thank you.

File: _data/icons/ebtc.json [hunk 2]
```diff
@@ -0,0 +1,8 @@
...
+    "height":690,
```
Comment (ligi) [inline _data/icons/ebtc.json:5]:
```suggestion
    "height":200,
```

File: _data/icons/ebtc.json (hunk 1 above)
Comment (ligi) [reply _data/icons/ebtc.json:4]:
```suggestion
    "width":200,
```

**Comment by BitcoinEvm** [comment]
https://github.com/ethereum-lists/chains/pull/1487 

New Pull Request Closing this

**Pull request closed without merging**